src/main.cpp

```diff
@@ -564,4 +566,5 @@ void exit_handler(int s)
 
         exit( exit_status );
     }
+    inp_mngr.set_timeout( old_timeout );
```

Comment:
Originally proposed it because CTRL+C leaves a black rectangle underneath, but CTRL+C is enough of an edge case not to bother.

Only tried in-game, not the main menu, so yeah, _bad idea_.